### PR TITLE
virttest.utils_test/.qemu: Add new class MultihostMigrationRdma

### PR DESCRIPTION
**Pull request by spcui**
ID: 1215548

### CODE REVIEW

File: virttest/utils_test/qemu.py [hunk 1]
```diff
@@ -1154,6 +1152,68 @@ def migrate_wait(self, vms_name, srchost, dsthost, start_work=None,
...
+        utils_misc.parallel(multi_mig)
+
+    def migrate_wait(self, vms_name, srchost, dsthost, start_work=None,
+                     check_work=None, mig_mode="rdma", params_append=None):
+
+        super_cls = super(MultihostMigrationRdma, self)
+        super_cls.migrate_wait(vms_name, srchost, dsthost,
+                               start_work=start_work, check_work=check_work,
+                               mig_mode=mig_mode, params_append=None)
+
```
Comment (ypu) [inline virttest/utils_test/qemu.py]: why we need rewrite this function?

Comment (spcui) [inline virttest/utils_test/qemu.py]: want to use mig_mode="rdma"

**Comment by ypu** [comment]
LGTM

Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by xutian** [comment]
Merged. thanks  @spcui  @ypu  and @suqinhuang

**Pull request merged**